local-nginx.conf

```diff
@@ -0,0 +1,34 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    location /api/search/ {
+        proxy_pass https://search.maven.org/;
+    }
+
+    location /api/ossindex/ {
+        proxy_pass  https://ossindex.sonatype.org/;
+    }
+
+    location /api/2/ {
+        proxy_pass  http://ww.google.com/;
+    }
+
+    location / {
+        root /usr/share/nginx/html;
+        index index.html index.htm;
+        expires -1;
+        add_header Pragma "no-cache";
+        add_header Cache-Control "no-store, no-cache, must-revalidate, post-check=0, pre-check=0";
+        try_files $uri$args $uri$args/ $uri $uri/ /index.html =404;
+    }
+
+    #error_page  404              /404.html;
```

Comment:
We actually take 404s and send them back to index.html (this allows the Angular router to attempt to load URLs that wouldn't typically load like: https://search.maven.org/artifact/org.robolectric/junit/4.0-beta-1/jar (since this wouldn't load the index.html page)

Reply:
seems logical, will do